drivers/memc/Kconfig

```
@@ -13,7 +13,7 @@ if MEMC
 
 config MEMC_INIT_PRIORITY
 	int "Initialization priority"
-	default 0
+	default 80 if MSPI
```

This is the reason why CI failed. LOL
```suggestion
	default 80 if MSPI
	default 0
```